### PR TITLE
Add performance metric of tikv-br

### PR DESCRIPTION
**Pull request by haojinming**
Signed-off-by: haojinming <jinming.hao@pingcap.com>

### CODE REVIEW

File: content/docs/dev/concepts/explore-tikv-features/backup-restore.md [hunk 1]
```diff
@@ -163,4 +163,26 @@ Please note that if data is stored in TiKV with [TTL](../ttl), and expiration ha
...
+### Performance
+
+The backup and restoration are both distributed. So the backup and restoration performance can increase linearly with the TiKV nodes' count if the storage and network don't reach the limits. The following are some key metrics of TiKV-BR backup and restoration for reference.
+- TiKV node: `4C8G`
+- PD node: `4C8G`
+- TiKV-BR node: `4C8G`
+
+|Metric|v6.4 apiv1|v6.4 apiv2|
+|:-:|:-:|:-:|
+|Max storage size|50T|50T|
```
[the remark this answers is not in the document]
Comment (haojinming) [reply content/docs/dev/concepts/explore-tikv-features/backup-restore.md]: Ok changed, PTAL~ thanks.

File: content/docs/dev/concepts/explore-tikv-features/backup-restore.md [hunk 2]
```diff
@@ -163,4 +163,26 @@ Please note that if data is stored in TiKV with [TTL](../ttl), and expiration ha
...
+|Backup speed|40MB/s per TiKV node|40MB/s per TiKV node|
+|Restoration speed|70MB/s per TiKV node|70MB/s per TiKV node|
+|Performance impact|20% on QPS/Latency|20% on QPS/Latency|
+
+#### Performance tuning
+
+If you want to reduce the impact of backup tasks on the cluster, you can enable the [`auto-tune`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#enable-auto-tune-%E4%BB%8E-v54-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) feature. With this feature enabled, TiKV-BR performs backup tasks as fast as possible without excessively affecting the cluster.
+
+Alternatively, you can limit the backup speed by using the TiKV configuration item [`backup.num-threads`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#num-threads-1) or using the parameter `--ratelimit`.
```
[the remark this answers is not in the document]
Comment (haojinming) [reply content/docs/dev/concepts/explore-tikv-features/backup-restore.md]: Addressed, thanks for the suggestions. PTAL~

**Review by pingyu — APPROVED** [review]
LGTM~

**Comment by pingyu** [comment]
@zeminzhou PTAL, thanks~

**Review by sunxiaoguang — APPROVED** [review]
LGTM

**Pull request merged**